cli/core/pkg/artifact/http.go

```diff
@@ -72,3 +74,8 @@ func (g *HTTPArtifact) Fetch() ([]byte, error) {
 
 	return out, nil
 }
+
+// FetchTest returns test artifact
+func (g *HTTPArtifact) FetchTest() ([]byte, error) {
+	return nil, errors.New("fetching test plugin from HTTP source is not yet supported")
```

Comment:
I feel it would be better to use `errors` package consistently across all the files. Most of the other places we are using `errors.New`, `errors.Wrap`, `errors.Wrapf` to handle the error.

Reply:
Now that `fmt.Errorf` supports wrapping with `%w`, I don't feel `pkg/errors` is necessary anymore.

Reply:
Thanks for the feedback. This is good to know. 

But if you don't mind, I would prefer to do that change as a separate PR and not hold this PR for that.